### DISCUSSION TITLE
Welcome to playwright-slack-report Discussions!

### DISCUSSION THREAD

**Discussion by ryanrosello-og** [ryanrosello-og/playwright-slack-report#102] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by soni26pro** [comment]
Will these work if I am using playwright with cucumber-js ?

`const getWorldParams = () => {
  const params = {
    foo: 'bar',
  };

  return params;
};

const config = {
  requireModule: ['ts-node/register'],
  require: ['src/**/*.ts'],
  format: [
    'json:reports/cucumber-report.json',
    `html:reports/${process.env.ENV}.html`,
    'progress-bar',
  ],
  formatOptions: { snippetInterface: 'async-await' },
  worldParameters: getWorldParams(),
};

// Check if USE_ALLURE environment variable is set
if (process.env.USE_ALLURE) {
  config.format.push('./src/support/reporters/allure-reporter.ts');
} else {
  config.format.push('@cucumber/pretty-formatter');
}

// Add playwright-slack-report if needed
if (process.env.USE_PLAYWRIGHT_SLACK_REPORTER) {
  config.format.push([
    './node_modules/playwright-slack-report/dist/src/SlackReporter.js',
    {
      sendResults: 'always', // "always" , "on-failure", "off"
      channels: ['external-services-for-uptime'], // provide one or more Slack channels
    },
  ]);
}

export default config;
`   I gave a try but end up with and error
